xml/System/Int128.xml

```diff
@@ -291,7 +291,7 @@
         <Parameter Name="value" Type="System.Int128" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
+        <param name="other">An object to compare with this instance.</param>
```

Comment:
I believe so, because the "returns" section references the "other" parameter. I'm realizing now that it would need to be updated on 291 as well, right?

Reply:
Interesting - in this case, [it looks like the returns section is incorrect](https://github.com/dotnet/runtime/blob/af1e6ba6414d5794d751868c9762dad2e95e8fc5/src/libraries/System.Private.CoreLib/src/System/Int128.cs#L47). We should look for other instances of this problem.

I'm also curious why the build system didn't pick up on the bad `<paramref>` - @kexugit FYI.

Reply:
I logged https://dev.azure.com/ceapex/Engineering/_workitems/edit/709950 about the lack of a warning in this case.

Reply:
Was the parameter ever renamed?

Reply:
I wrote too soon, I see the problem. No need to update the param name, that's autogenerated, plus it's the actual name of the parameter in the child API, even if the parent API has another name.

Reply:
Resolved with https://github.com/dotnet/dotnet-api-docs/pull/8403/commits/f73a26d6540f3daa2baf95b556266306476d1025#